deis-workflow-manager/README.md

```diff
@@ -0,0 +1,7 @@
+# deis-workflow-manager
+
+This chart manages the deis-workflow-manager Deis component. Exposes a service
+listening port on TCP 8443, and a matching listening port on the RC container.
```

Comment:
The wording of this feels a bit odd, but rather than fix it, I wonder if maybe some of the detail can be sacrificed instead.